source/audio/soundSplit.py

```diff
@@ -15,6 +15,7 @@
 import ui
 from utils.displayString import DisplayStringIntEnum
 from dataclasses import dataclass
+import _ctypes

```

Comment:
usually we import this from `comtypes`

Note `isinstance(comtypes.COMError(1, 2, 3), _ctypes.COMError)` is True.

```suggestion
from comtypes import COMError
```